app/points/calculate-points-for-workload.js

```diff
@@ -1,13 +1,39 @@
 const calculatePointsForTiers = require('../../app/points/calculate-points-for-tiers')
+const calculateSdrConversionPoints = require('../../app/points/calculate-sdr-conversion-points')
+const calculateParomPoints = require('../../app/points/calculate-parom-points')
+const calculateArmsPoints = require('../../app/points/calculate-arms-points')
+
 const Workload = require('../../app/points/domain/workload')
 const CaseTypeWeightings = require('../../app/points/domain/case-type-weightings')
 const assertObjectType = require('../../app/points/domain/validation/assert-object-type')
 
 module.exports = function (workload, caseTypeWeightings) {
   assertObjectType(workload, Workload, 'workload')
   assertObjectType(caseTypeWeightings, CaseTypeWeightings, 'CaseTypeWeightings')
-  var totalWorkloadPoints = calculatePointsForTiers(workload.communityTiers, caseTypeWeightings.pointsConfiguration.communityTierPointsConfig, caseTypeWeightings) +
-                            calculatePointsForTiers(workload.custodyTiers, caseTypeWeightings.pointsConfiguration.custodyTierPointsConfig, caseTypeWeightings) +
-                            calculatePointsForTiers(workload.licenseTiers, caseTypeWeightings.pointsConfiguration.licenseTierPointsConfig, caseTypeWeightings)
-  return totalWorkloadPoints
+
+  var communityTierPoints = calculatePointsForTiers(workload.communityTiers, caseTypeWeightings.pointsConfiguration.communityTierPointsConfig, caseTypeWeightings)
+  var custodyTierPoints = calculatePointsForTiers(workload.custodyTiers, caseTypeWeightings.pointsConfiguration.custodyTierPointsConfig, caseTypeWeightings)
+  var licenseTierPoints = calculatePointsForTiers(workload.licenseTiers, caseTypeWeightings.pointsConfiguration.licenseTierPointsConfig, caseTypeWeightings)
+
+  var sdrConversionPointsLast30Days = calculateSdrConversionPoints(workload.sdrConversionsLast30Days, caseTypeWeightings.pointsConfiguration.sdrConversion)
+  var monthlySdrConversionPoints = calculateSdrConversionPoints(workload.monthlySdrs, caseTypeWeightings.pointsConfiguration.sdr)
+  var paromsPoints = calculateParomPoints(workload.paromsCompletedLast30Days, caseTypeWeightings.pointsConfiguration.parom, caseTypeWeightings.pointsConfiguration.paromsEnabled)
+  var armsPoints = calculateArmsPoints(workload.armsLicenseCases, workload.armsCommunityCases, caseTypeWeightings.armsLicense, caseTypeWeightings.armsCommunity)
+
+  var totalWorkloadPoints = communityTierPoints +
+                                custodyTierPoints +
+                                licenseTierPoints +
+                                sdrConversionPointsLast30Days +
+                                monthlySdrConversionPoints +
+                                paromsPoints +
+                                armsPoints
```

Comment:
So this all points calculations are being moved from worker into probation rules (sdrConversionPoints, sdrPoints, paromsPoints)?  In worker it will now just read `totalPoints = workloadPoints + adjustments(CMS and GS)`